### PR TITLE
quic: updating test comment

### PR DESCRIPTION
**Pull request by alyssawilk**

### CODE REVIEW

**Comment by alyssawilk** [comment]
cc @danzh2010

File: test/integration/tcp_tunneling_integration_test.cc [hunk 1]
```diff
@@ -743,7 +743,10 @@ TEST_P(TcpTunnelingIntegrationTest, TcpProxyDownstreamFlush) {
 // Test that an upstream flush works correctly (all data is flushed)
 TEST_P(TcpTunnelingIntegrationTest, TcpProxyUpstreamFlush) {
   if (upstreamProtocol() == Http::CodecType::HTTP3) {
-    // TODO(alyssawilk) debug.
+    // The payload data depends on having TCP buffers upstream and downstream.
+    // For HTTP/3, upstream, the flow control window will back up sooner, Envoy
```
Comment (danzh2010) [inline test/integration/tcp_tunneling_integration_test.cc:747]: Interesting, is it because the default QUICHE flow control window we use is different?

Comment (alyssawilk) [reply test/integration/tcp_tunneling_integration_test.cc:747]: different than the kernel socket buffer size, yeah.  I'm inclined to think unifying them is not worth the trouble for this test.

**Review by davinci26 — APPROVED** [review]
lgtm, thanks for explanation

**Pull request merged**